plugins/flytekit-dbt/requirements.in

```diff
@@ -1,5 +1,5 @@
 .
 -e file:.#egg=flytekitplugins-dbt
 # dbt-sqlite and dbt-core should be compatible
-dbt-core==1.2.1
+dbt-core==1.4.1
 dbt-sqlite==1.2.0a2
```

Comment:
we should remove `dbt-core` and just list `dbt-sqlite` in this file (this should bring a version of `dbt-core` that's compatible).